### PR TITLE
Add notebook for data handling of kaggle dataset

### PR DESCRIPTION
**Pull request by sooahleex**
### Summary

- Add notebook to show how to clean dataset from kaggle through Datumaro
- This PR should be merged after #1533

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [X] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: notebooks/21_kaggle_data_cleaning.ipynb [hunk 1]
```diff
@@ -0,0 +1,642 @@
+{
```
Comment (wonjuleee) [inline notebooks/21_kaggle_data_cleaning.ipynb:1]: <p>can we try this with dm_dataset[0] instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1534/discussion/#comment-1652152315'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (wonjuleee) [inline notebooks/21_kaggle_data_cleaning.ipynb:1]: <p>could you use <code>dm_dataset[0].media.data</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1534/discussion/#comment-1652152320'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='26'/>

Comment (wonjuleee) [inline notebooks/21_kaggle_data_cleaning.ipynb:1]: <p>I am sorry that I was confused about the cleaned example.</p><p>Is this expected to remove <code>I, but I, doing it, etc.. </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1534/discussion/#comment-1652152323'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='33'/>

Comment (sooahleex) [reply notebooks/21_kaggle_data_cleaning.ipynb:1]: Fix in [3c2d3c1](https://github.com/openvinotoolkit/datumaro/pull/1534/commits/3c2d3c1b0b0d0f7457dabdd90398dd8a00bb04e9)

Comment (sooahleex) [reply notebooks/21_kaggle_data_cleaning.ipynb:1]: Fix in [3c2d3c1](https://github.com/openvinotoolkit/datumaro/pull/1534/commits/3c2d3c1b0b0d0f7457dabdd90398dd8a00bb04e9)

Comment (sooahleex) [reply notebooks/21_kaggle_data_cleaning.ipynb:1]: This [link](https://gist.github.com/sebleier/554280) is the list of stopwords, and in our clean transformation we remove html tags, urls, emoticons, special characters and punctuation. So in this case the result is removed stopwords and special characters. If you think this index is not appropriate to the example, I will change the number of index.

**Review by wonjuleee — APPROVED** [review]

**Pull request merged**